tests/test_atom_mapping.py

```diff
@@ -1032,6 +1025,7 @@ def go(k):
 def make_polyphenylene(n, dihedral_deg):
     """Make a chain of n benzene rings with each ring rotated `dihedral_deg` degrees with respect to the previous ring"""
     mol = Chem.MolFromSmiles(polyphenylene_smiles(n))
+    mol.SetProp("_Name", f"{n}_{dihedral_deg}")
```

Comment:
Added a name so that I could plot these. If you change the version of RDKit the tests that use these mols fail due to a change the output of `AllChem.EmbedMolecule`

Reply:
Worth adding `set_mol_name` to `timemachine.fe.utils` to complement `get_mol_name`?

Reply:
Added in https://github.com/proteneer/timemachine/pull/1362/commits/17f22fc9c050f52732adcf81fe00fbbfb2dbd14d